templates/.travis.yml

```diff
@@ -2,10 +2,9 @@ sudo: false
 
 language: node_js
 node_js:
-  - stable
-
+  - '9'
+  - '8'
```

Comment:
just as  a tangent, are you doing this because bfly is not on 8? because if that's the case, we should update probably?

Reply:
we have a ticket to upgrade to 10

Reply:
it is basically a painless upgrade. but we should test this on 10?

Reply:
its not painless plenty of ecosystem stuff is still broken :/ i've had to pin to 9 on most stuff. I did 8 here as well since it's good to test against the real stable, and next